themes/hugo-theme-altinn/layouts/partials/header.html

```
@@ -83,8 +83,12 @@
               <a class="label label-default" href="{{$.Site.BaseURL}}tags/{{ $tag | urlize }}">{{ $tag }}</a>
             {{- end -}}
           </div>
-        {{- end -}}        
-        <div id="body-inner">
+        {{- end -}}
+        {{- if and true .Params.disablepagefind -}}
+          <div id="body-inner">
+        {{- else -}}
+          <div id="body-inner" data-pagefind-body>
```

HTMLHint (1.5.0)

[error] The id value [ body-inner ] must be unique.

(id-unique)